flask_restful/fields.py

```diff
@@ -415,3 +421,24 @@ def _iso8601(dt):
     :return: A ISO 8601 formatted date string
     """
     return dt.isoformat()
+
+
+def _rfc3339(dt):
+    """Turn a datetime object into an RFC3339 formatted date at UTC.
+
+    Example::
+
+        fields._rfc3339(datetime(2012, 1, 1, 0, 0, tzinfo=pytz.utc)) => "2012-01-01T00:00:00Z"
+
+    :param dt: The datetime to transform
+    :type dt: datetime
+    :return: A RFC 3339 formatted date string
+    """
+    if dt.tzinfo is None or dt.tzinfo.utcoffset(dt) is None:
+        raise MarshallingException(
+            'Naive datetime %s not supported by RFC 3339.' % dt
+        )
+    d = dt.astimezone(pytz.utc)
+    ts = timegm(d.timetuple())
```

Comment:
I'd be very cautious about this. I think this process converts timezone offsets to UTC (see #405).